### PR TITLE
Send license usage

### PR DESCRIPTION
**Pull request by timgl**
## Changes

Each day, send usage of the entire instance to the valid license (if applicable) for automatic billing purposes.

Requires https://github.com/PostHog/license/pull/16 to be merged first.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

File: ee/tasks/send_license_usage.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from dateutil.relativedelta import relativedelta
+from django.utils import timezone
+
+from ee.clickhouse.client import sync_execute
+from ee.models.license import License
+from posthog.models import User
+
+
+def send_license_usage():
+    license = License.objects.first_valid()
```
Comment (paolodamico) [inline ee/tasks/send_license_usage.py:12]: Should this be `first_valid`? Maybe we report on all valid licenses?

File: posthog/celery.py [hunk 2]
```diff
@@ -260,6 +264,14 @@ def clickhouse_mark_all_materialized():
         mark_all_materialized()
 
 
+@app.task(ignore_result=True)
+def clickhouse_send_license_usage():
+    if is_clickhouse_enabled() and settings.EE_AVAILABLE:
```
Comment (paolodamico) [inline posthog/celery.py:269]:
```suggestion
    if is_clickhouse_enabled():
```

`is_clickhouse_enabled` already checks `settings.EE_AVAILABLE`

**Review by paolodamico — APPROVED** [review]
lgtm!

File: ee/tasks/send_license_usage.py (hunk 1 above)
Comment (timgl) [reply ee/tasks/send_license_usage.py:12]: Hm so everything else that we do (like checking what plan a user has access to) is all done on first_valid as well so this feels right. People should really only have one valid License enabled anyway.

**Pull request merged**